Solutions/MailRisk/Data Connectors/azuredeploy_SecurePractice_MailRisk.json

```diff
@@ -228,7 +222,7 @@
             "type": "Microsoft.Web/sites",
             "apiVersion": "2022-03-01",
             "name": "[variables('FunctionName')]",
-            "location": "[parameters('location')]",
+            "location": "[variables('location')]",
             "dependsOn": [
                 "[resourceId('Microsoft.Storage/storageAccounts', tolower(variables('FunctionName')))]",
                 "[resourceId('Microsoft.Insights/components', variables('FunctionName'))]",
```

Comment:
@ingin97 : To fix **functions runtime is unreachable error** please modify below 2 sections at mentioned line numbers.

1. Modify line# 231 from: **"kind": "functionapp", ** To:  **"kind": "functionapp,linux", **
2. Append below section at line@ 240 under properties, 
               **"siteConfig": {
                                  "linuxFxVersion": "python|3.8"
                              }**